settings/Privacy.qml

```diff
@@ -108,9 +105,9 @@ Page {
                 anchors.horizontalCenter: parent.horizontalCenter
 
                 onClicked: {
-                    //: Remorse item for clearing private date
-                    //% "Clearing"
-                    clearDataRemorse.execute(qsTrId("settings_browser-la-clearing_private_data"),
+                    //: Remorse item for clearing private data
+                    //% "Cleared"
+                    Remorse.popupAction(page, qsTrId("settings_browser-la-cleared_private_data"),
```

Comment:
Agree though fixing it would basically mean re-implementing privacy clearing, which I think is out of scope of the pull request here, especially with the given schedule pressure.

Reply:
Can live with it, i suppose, but can also change the deleted things while the remorse is ticking, and can click the button too though it doesn't do anything at that point :)